src/compiler/checker.ts

```diff
@@ -14771,6 +14771,49 @@ namespace ts {
             }
         }
 
+        // Static members may conflict with non-configurable non-writable built-in Function object properties 
+        // see https://github.com/microsoft/typescript/issues/442.

```

Comment:
instead of a github issue number, can you reference the spec the way you do below instead?